### PR TITLE
Auto-update image preview, add label, and set max size

### PR DESCRIPTION
**Pull request by lmcnulty**
Solves #635. Shows fallback image by default, also resulting in image preview updating as url is typed rather than after focus exits in the input. Sets a max-height on the preview and adds a label:

image

### CODE REVIEW

**Review by cesarvarela — COMMENTED** [review]
I think the issue is somewhere else because the fallback handling is done here: https://github.com/responsible-ai-collaborative/aiid/blob/fba1626eceeaeac22e5fba1c25b4a82e76a5304b/site/gatsby-site/src/utils/cloudinary.js#L26.

Looks more like a render cycle problem, but maybe it is not a rabbit hole worth getting into.

I would like to see a couple of tests checking each outcome of this change though (the `integrations/submit.js` page seems a good candidate).

**Comment by smcgregor** [comment]
LGTM

@cesarvarela : you have been spending time in the submit testing code. Can you take final review on this one?

**Comment by smcgregor** [comment]
@lmcnulty can you resolve the merge conflict then ping @cesarvarela for final review?

File: site/gatsby-site/cypress/integration/submit.js [hunk 1]
```diff
@@ -486,6 +486,29 @@ describe('The Submit form', () => {
...
+
+    const newImageUrl = 'https://' + suffix;
+
+    const cloudinaryImageUrl =
+      'https://res.cloudinary.com/pai/image/upload/d_fallback.jpg/f_auto/q_auto/v1/reports/' +
+      suffix;
+
+    cy.visit(url);
+    cy.get('input[name=image_url]').type(newImageUrl);
+    cy.wait(1000);
```
Comment (cesarvarela) [inline site/gatsby-site/cypress/integration/submit.js:508]: This `wait` might introduce some flakiness, have you tried something else? Especially on CI where everything runs slower. 

I'm assuming the test fails without it?

(If there is no better way let's go with it)

**Review by cesarvarela — APPROVED** [review]

File: site/gatsby-site/cypress/integration/submit.js (hunk 1 above)
Comment (lmcnulty) [reply site/gatsby-site/cypress/integration/submit.js]: Ah! Good catch. That was leftover from a previous state and is no longer necessary. I've removed it in the last commit.

**Comment by smcgregor** [comment]
I started the deploy preview and will merge if it passes.

**Pull request merged**